include/edhoc_consts.h

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Same comment as for `edhoc_rs.h`. Why do you include this in the repo when it should be generated on the fly?

Reply:
Same as above.